ext/nnstreamer/tensor_filter/tensor_filter_tvm.cc

```diff
@@ -162,7 +161,15 @@ tvm_subplugin::parse_custom_prop (const char *custom_prop)
           nns_loge ("Unknown device (%s).", option[1]);
           invalid_option = true;
         }
-      } else {
+      } else if (g_ascii_strcasecmp (option[0], "num_input") == 0) {
+        inputInfo.num_tensors = MIN ( atoi (option[1]), NNS_TENSOR_SIZE_LIMIT );
```

Comment:
prevent security report later

recommend to use g_ascii_strtoull() (atoi() is obsolescent function)
see https://wiki.sei.cmu.edu/confluence/display/c/MSC24-C.+Do+not+use+deprecated+or+obsolescent+functions

```
(guint) g_ascii_strtoull (option[1], NULL, 10);
```

Reply:
Fixed!! Thanks for the info